src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.cs

```diff
@@ -427,29 +422,56 @@ private void NotifyRenameEventArgs(WatcherChangeTypes action, string name, strin
             }
         }
 
-        /// <devdoc>
-        ///     Raises the event to each handler in the list.
-        /// </devdoc>
-        /// <internalonly/>
-        private void NotifyFileSystemEventArgs(WatcherChangeTypes changeType, string name)
+        /// <summary>
+        /// Raises the event to each handler in the list.
+        /// </summary>
+        private void NotifyRenameEventArgs(WatcherChangeTypes action, ReadOnlySpan<char> name, ReadOnlySpan<char> oldName)
```

Comment:
Yes, this is for the Unix builds. I can move it over there. I'm handing the Unix part over to @Anipik as a follow-up issue.

Reply:
Ok. Thanks.

Reply:
I will do the same thing for unix really soom

Reply:
I forgot that there are three different Unix implementations. There isn't a base class for all Unix- rather than introduce one I'm going to leave this as is for now as @Anipik is following up.